### DISCUSSION TITLE
Showing up the data from tables related at table layout options

### DISCUSSION THREAD

**Discussion by Sam-singtecs** [directus/directus#19961] [Feature Requests]
### Summary

Hi, so currently I have three tables (Time slots, Users and Bookings). Relationships are as follows:

- Timeslots has one to many relationship with Bookings
- Bookings has one to many relationship with Users

So in calendar view of Bookings, what I want to show up is the username from Users and date from Time slots tables.

Thank you in advance!

### Motivation

This calendar layout could be used as the same way as booking system. With more customization and display template, this layout has potential to be a complete booking system.

### Detailed Design

- Add child/sibling tables' column which have relationship in Calendar Layout options -> Display Template
- There should be a checkbox which decides **Start Date Field** to be shown in calendar or not

### Must Have:
-  Display Template in table layout options

### Should Have:
-  Showing time based on Timezone selection

### Drawbacks

It could potentially confuse the end users without proper explanation.

### Alternatives

None

### Adoption Strategy

None

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
